lib/response.js

```diff
@@ -520,5 +522,12 @@ module.exports = {
       message: this.message,
       header: this.header
     };
+  },
+
+  /**
+   * Flush any set headers, and begin the body
+   */
+  flushHeaders: function(){
+    this.res.write('');
```

Comment:
@PlasmaPower https://github.com/nodejs/node/blob/v5.7.0/lib/_http_server.js#L190

The error is that you are not providing the first argument, `statusCode`, to the `writeHead` method.